build_projects/dotnet-host-build/PublishTargets.cs

```diff
@@ -280,6 +281,23 @@ public static BuildTargetResult PublishCoreHostPackages(BuildTargetContext c)
         }
 
         [Target]
+        public static BuildTargetResult PublishSignedPackages(BuildTargetContext c)
+        {
+            string signedPackages = EnvVars.EnsureVariable("SIGNED_PACKAGES");
+            if (signedPackages.Equals("true"))
+            {
+                foreach (var file in Directory.GetFiles(Dirs.Packages, "*.nupkg"))
```

Comment:
Presumably, this only affects the two packages and not the existing host packages, right?